packages/renderer/src/stores/event-store.ts

```diff
@@ -263,6 +265,20 @@ export class EventStore<T> {
       });
     });
 
+    if (this.configurationSections) {
+      window.events?.receive('configuration-changed', (args?: unknown) => {
+        if (
+          this.configurationSections &&
+          isConfigurationChangedArg(args) &&
+          this.configurationSections.includes(args.section)
+        ) {
+          update('configuration-changed', [args]).catch((error: unknown) => {
+            console.error(`Failed to update ${this.name}`, error);
+          });
+        }
+      });
+    }
+
```

Comment:
I would favor to reuse an existing pattern from `windowEvents`

so instead of `configuration-changed` I can specify for example `configuration-changed:<key>`


`configuration-changed:extensions.ignoreRecommendations`

Reply:
so if other functions have keys or specific value it could be more generic rather than adding a new entry just to handle a specific event

it would be `NameOfEvent:Key`

Reply:
I think the later one

```
'NameOfEvent', { key: 'Key', ... }
```

if I take

```
 this.apiSender.send('configuration-changed', eventArg);
```

if there is a `eventKey` or `key` property inside the event, then I can use nameOfTheEvent:key to be triggered only if key is received

Reply:
OK, I like this pattern, I will implement it.
Thanks